data/tlg4021/tlg002/tlg4021.tlg002.opp-grc1.xml

```diff
@@ -100,3594 +104,6 @@ License</p>
-οἶδε. καὶ πάλιν ὁ γραμματικὸς λαμβάνει ὁμολογούμενον ὅτι τὸ ἡ
-καὶ τὸ #x772; μακρά εἰσι· τούτου δὲ τὴν αἰτίαν ὁ μουσικὸς φιλόσοφος οἶδεν.
-<lb n="20"/> ἡ φιλοσοφία οὖν, ὡς εἴρηται, καὶ ὀνομάζεται καὶ ὑφέστηκε καὶ ἐν πρώτοις
-τὴν ὕπαρξιν· ἔχει. καὶ ἐπειδὴ μὲν ὀνομάζεται, ἔχει ἐκ τοῦ ὀνόματος ὅρον
-τὸν λέγοντα ‘φιλοσοφίκ ἐστὶ φιλία σοφίας, ἐπειδὴ δὲ ἐν πρώτοις ἔχει τὴν
-ὕπαρξιν, ἔχει τὸν ἐκ τῆς ὑπεροχῆς ὅρον τὸν λέγοντα ‘φιλοσοφία ἐστὶ τέχνη τεχνῶν
-καὶ ἐπιστήμη ἐπιστημῶν’. ἐπειδὴ δὲ ὑφέστηκεν, ἔχει καὶ ὑποκείμενον, ἔχει καὶ
-<lb n="25"/> τέλος, καὶ ἑκάτερον τούτων διττόν· πάσα γὰρ τέχνη καὶ ἐπιστήμη ἔχει καὶ ὑποκείμενον,
-κείμενον, ἔχει καὶ τέλος, καὶ ἑκάτερον τούτων διττόν· ἔχει γὰρ προσεχὲς
-ὑποκείμενον, ἔχει καὶ πόρρω ὑποκείμενον, καὶ πάλιν ἔχει καὶ προσεχὲς τέλος
-καὶ πόρρω τέλος. ἀλλ’ ἵνα σαφὲς ἡμῖν τὸ λεγόμενον γένηται, γυμνάσωμεν τὸν
-λόγον ἐπὶ τῆς ναυπηγικῆς. ἰστέον ὅτι ἡ ναυπηγικὴ ἔχει καὶ ὑποκείμενον,
-<lb n="30"/> ἔχει καὶ τέλος, καὶ ἑκάτερον τούτων διττόν· ἔχει γὰρ καὶ προσεχὲς ὑποκείμενον
-καὶ πόρρω ὑποκείμενον, καὶ πάλιν ἔχει προσεχὲς τέλος καὶ πόρρω
-τέλος. καὶ προσεχὲς μὲν ὑποκείμενον ἔχει τὰ ἁπλῶς ξύλα, πόρρω δὲ
-ὑποκείμενον τὰ τοιάδε ξύλα οἷον τὰ πεποιωμένα πρὸς τρόπιν καὶ πηδάλιον·
-πρῶτον γὰρ ὁ ναυπηγὸς λαμβάνει τὰ ἁπλῶς ξύλα, καὶ εἶθ’ οὕτως ζητεῖ
-<lb n="35"/> ποῖα πεποίωνται πρὸς τρόπιν καὶ πηδάλιον. καὶ πάλιν προσεχὲς μὲν τέλος
-<note type="footnote">5 pr. τῶ om. T 6 ἅτινα] ἁ T ὑφ’] παρ’ T 7 ὑφιστάμενα T
-9 τινα om. KV 10 ante ὁ ἄνθp. add. καὶ KV ante ὀνομ. add. καὶ T
-11 ἐν τῷ εἰ ἔστι] cf. p. 1,15 15. 16 ὁ ἰατp. δὲ πάλιν καὶ om.) T 17 συνίστ. σώμ.
-coll. T συνίσταται T 17. 18 οἶδε φιλ. coll. KV 18 λαμβάνει om. KV
-post ὅτι add. καὶ T 21. 22 τὸν ὅρον τὸν T 24 alt. ἔχει om. T 25 διττόν]
-διπλοῦν T 25. 26 πᾶσα—διττόν om. T 26 pr. ἔχει om. Κ 27 καὶ (ante
-προσ.) om. T 28 γέν. τὸ λεγ. coll. T 29 τῆς ναυπ.] παραδείγματος T 30 ἔχει
-(post ὐποκ.) om. T 31 ὑποκ. (post πόρρω) om. T 32 τέλος om. T 33 πεποιωμένα
-scripsi : πεποιημένα libri τροπὴν T 35 πεποίωνται scripsi : πεποίηνται libri</note>
-
-<pb n="22"/>
-ἔχει τὸ ποιῆσαι ἁπλῶς ναῦν, πόρρω δὲ τέλος τὸ ποιῆσαι τοιάνδε νῦν, οἷον
-ἢ δόρκωνα ἢ λύχνον. οὕτως οὖν καὶ ἡ φιλοσοφία ἔχει καὶ ὑποκείμενον
-καὶ τέλος καὶ ἑκάτερον τούτων διττόν· ἔχει γὰρ προσεχὲς ὑποκείμενον καὶ
-πόρρω καὶ τέλος ὁμοίως προσεχὲς καὶ πόρρω. καὶ προσεχὲς μὲν ὑποκεί-
-<lb n="5"/> μένον ἔχει τὰ ἁπλῶς ὄντα, πόρρω δὲ ὑποκείμενον τὰ τοιάδε ὄντα, οἷον τὰ
-θεῖα καὶ τὰ ἀνθρώπινα πράγματα. καὶ πάλιν προσεχὲς μὲν τέλος ἔχει τὸ
-μελετᾶν τὸν θάνατον, ἤγουν τὸ νέκρωσιν ἀπεργάζεσθαι τῶν παθῶν, πόρρω
-δὲ τέλος ἔχει | τὸ διὰ τούτου, ἤγουν διὰ τοῦ νέκρωσιν ἀπεργάζεσθαι τῶν <note type="marginal">137r</note>
-παθῶν, ὁμοιοῦσθαι θεῷ κατὰ τὸ δυνατὸν ἀνθρώπῳ. ἐπειδὴ οὖν ἡ φιλοσοφία
-<lb n="10"/> ἔχει καὶ ὑποκείμενον καὶ τέλος καὶ ἑκάτερον τούτων διττόν, ὡς εἴρηται,
-τούτου χάριν καὶ ἄλλους τέσσαρας ὅρους ἔχει, δύο ἐκ τοὐ ὑποκειμένου, ἕνα
-μὲν ἐκ τοῦ προσεχοῦς ὑποκειμένου τὸν λέγοντα ὅτι ‘φι
```

Comment:
This should probably be changed to `n="1"` instead of `n="2"`, since they are two different works and since we are already moving away from the edition in that we are separating two works that it combined.

Reply:
I'm afraid @sonofmun I don't quite understand this change request, as there is no 2, in the example you show, to change to 1. Did you mean that at the start of the new tlg4021.tlg002 file, I should change 

```xml
<div type="textpart" subtype="chapter" n="2">
<pb n="80"/>
```
n="2 to n="1" there?  Or am I entirely on the wrong page here?